app/lib/nbtc.ts

```diff
@@ -1,4 +1,146 @@
+import { networks, opcodes, Psbt, script, type Network } from "bitcoinjs-lib";
+import Wallet from "sats-connect";
+import type { Address, RpcResult } from "sats-connect";
+import { fetchUTXOs, fetchValidateAddress } from "~/api/btcrpc";
+import type { UTXO, ValidateAddressI } from "~/api/btcrpc";
+import type { ToastFunction } from "~/hooks/use-toast";
+import mintTestNetConfig from "~/config/mint/contracts-testnet.json";
+import mintMainNetConfig from "~/config/mint/contracts-mainnet.json";
+import { isMainNetNetwork } from "./appenv";
+
 export const PRICE_PER_NBTC_IN_SUI = 25000n;
-export const nBTC_ADDR = "tb1qe60n447jylrxa96y6pfgy8pq6x9zafu09ky7cq";
 export const NBTC_COIN_TYPE =
 	"0x5419f6e223f18a9141e91a42286f2783eee27bf2667422c2100afc7b2296731b::nbtc::NBTC";
```

Comment:
this should be in the config. Create a task for it to fix it. Should be top priority

Reply:
yes, this is in config already.

Reply:
https://github.com/gonative-cc/byield/issues/324

Reply:
https://github.com/gonative-cc/byield/issues/324